src/api/handlers/auth/getToken.js

```diff
@@ -0,0 +1,41 @@
+import jwt from "jsonwebtoken";
+import { db } from "../../../../knex";
+import { jwtConfig } from "../../../config";
+
+/*
+ * Create a token for a user.
+ */
+export async function getToken(req, res, next) {
+  let user = null;
+
+  try {
+    user = await db("users")
+    .select("email", "password")
+    .where("email", req.body.email)
+    .first();
+  } catch (error) {
+    return res.json({ success: false, message: "Authentication failed. User not found." });
+  }
+
+  if (!user) {
+    return res.json({ success: false, message: "Authentication failed. User not found." });
+  }
+
+  if (user.password !== req.body.password) {
```

Comment:
We'll want to hash and salt the password when we store it. We could use bcrypt like this: https://www.abeautifulsite.net/hashing-passwords-with-nodejs-and-bcrypt